tests/uvdata/test_uvfits.py

```diff
@@ -168,6 +169,11 @@ def test_time_precision(tmp_path):
         atol=uvd2._lst_array.tols[1],
     )
 
+    if uvw_double:
+        np.testing.assert_allclose(uvd.uvw_array, uvd2.uvw_array, rtol=1e-13)
+    else:
+        np.testing.assert_allclose(uvd.uvw_array, uvd2.uvw_array, rtol=1e-7)
```

Comment:
@bhazelton -- I think actually holding on to both checks here would be good (both that it fails if `rtol=1e-13` and passes when `rtol=1e-7`), since that's a pretty strong indicator that double-precision _isn't_ being used. Or otherwise, adding a check that the UU2/VV2/WW2 entries aren't in there (although I'm not sure if there's a straight-forward way to do that w/ the astropy FITS interface).

Reply:
ok, added that one back.